stack/rest/src/main/java/org/apache/usergrid/rest/IndexResource.java

```diff
@@ -250,7 +252,8 @@ private ReIndexService getReIndexService() {
 
 
     private ReIndexRequestBuilder createRequest() {
-        return createRequest();
+        //TODO: wire this up through spring, and in the future guice.
+        return new ReIndexRequestBuilderImpl();
```

Comment:
This was an endless loop, and should be returning the builder this is making a call through to core skipping services. Maybe this needs a ticket associated with it?